Add documentation for Inngest Workflow (#4276)
mastra-ai/mastra: docs/src/content/en/examples/workflows_vNext/inngest-workflow.mdx

      },
    ])
 
    let activitiesText = ''
 
    for await (const chunk of response.textStream) {
      process.stdout.write(chunk)
      activitiesText += chunk
    }
 
    return {
      activities: activitiesText,
    }
  },
})

// Compose the workflow with conditional branching
const activityPlanningWorkflow = createWorkflow({
  id: 'activity-planning-workflow-step2-if-else',
  inputSchema: z.object({
    city: z.string().describe('The city to get the weather for'),
  }),
  outputSchema: z.object({
    activities: z.string(),
  }),
})
  .then(fetchWeather)
  .branch([
    [
      // If precipitation chance is greater than 50%, suggest indoor activities
      async ({ inputData }) => {
        return inputData?.precipitationChance > 50
      },
      planIndoorActivities,
    ],
    [
      // Otherwise, suggest a mix of activities
      async ({ inputData }) => {
        return inputData?.precipitationChance <= 50
      },
      planActivities,
    ],
  ])
 
activityPlanningWorkflow.commit()
 
export { activityPlanningWorkflow }
```

## Register Agent and Workflow instances with Mastra class

Register the agents and workflow with the mastra instance.

> This is critical for enabling access to the agents within the workflow.

Reviewer: ```suggestion
This allows access to the agents within the workflow.
```
